### ISSUE TITLE
`Error [network]: No such host is known. (os error 11001)` window 11

### ISSUE THREAD

**Issue by OmiChrome** [ogham/dog#118]
dog not working in window at all , 
`Error [network]: No such host is known. (os error 11001)`

- dog -v 
> v0.1.0

- window 11 , i3 11thgen vivobook

![image](https://user-images.githubusercontent.com/108952834/233467230-31ad4480-fd7e-4d5e-9a99-c1ba17d46953.png)

![image](https://user-images.githubusercontent.com/108952834/233467322-2d9747c0-6002-46f7-af0b-b0be4cb3d075.png)

some note : 
i installed it using [scoop ](scoop.sh)  [bucket](https://github.com/ScoopInstaller/Main/blob/master/bucket/dog.json)

**Comment by SeryiBaran** [comment]
same problem =(

**Comment by yitsushi** [comment]
The workaround under this issue works: https://github.com/ogham/dog/issues/9

tl;dr: it can't get the dns server, `-n DNSSERVER` it works
```
PS> dog efertone.me MX
Error [network]: No such host is known. (os error 11001)

PS> dog -n 1.1.1.1 efertone.me MX
MX efertone.me. 5m00s   10 "mail.protonmail.ch."
MX efertone.me. 5m00s   20 "mailsec.protonmail.ch."
```

I think this one can be closed as duplicate.
